modules/web_hosting/network.tf

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#########################################################################
+# vpc-xlb - VPC Network & Subnests
+#########################################################################
+
+resource "google_compute_network" "vpc-xlb" {
+  name                    = "vpc-xlb"
+  project                 = local.project.project_id
+  auto_create_subnetworks = "false"
+  routing_mode            = "GLOBAL"
+  depends_on              = [google_project_service.enabled_services]
+}
+
+# Creating Sunbet for vpc-xlb VPC network
+resource "google_compute_subnetwork" "subnetwork-vpc-xlb-us-c1" {
+  name                     = "vpc-xlb-us-c1"
+  ip_cidr_range            = "10.200.20.0/24"
+  region                   = "us-central1"
```

Comment:
Can we replace this with a variable?  Applies to most of these values.

Reply:
Done